packages/x-components/src/x-modules/facets-next/components/filters/__tests__/editable-number-range-filter.spec.ts

```diff
@@ -0,0 +1,343 @@
+import { EditableNumberRangeFilter, RangeValue } from '@empathyco/x-types-next';
+import { createLocalVue, mount, Wrapper } from '@vue/test-utils';
+import Vue from 'vue';
+import Vuex, { Store } from 'vuex';
+// eslint-disable-next-line max-len
+import { createNextEditableNumberRangeFilter } from '../../../../../__stubs__/filters-stubs.factory';
+import { getDataTestSelector, installNewXPlugin } from '../../../../../__tests__/utils';
+import {
+  getXComponentXModuleName,
+  isXComponent
+} from '../../../../../components/x-component.utils';
+import { XPlugin } from '../../../../../plugins/x-plugin';
+import { RootXStoreState } from '../../../../../store/store.types';
+import { DeepPartial } from '../../../../../utils/types';
+import { facetsNextXModule } from '../../../x-module';
+import { resetXFacetsStateWith } from '../../__tests__/utils';
+import EditableNumberRangeFilterComponent from '../editable-number-range-filter.vue';
+
+Object.defineProperty(HTMLInputElement.prototype, 'valueAsNumber', {
+  get() {
+    return parseFloat(this.value);
+  },
+  configurable: true,
+  enumerable: true
+});
+
+function renderEditableNumberRangeFilter({
+  template = `
+    <EditableNumberRangeFilterComponent
+      :filter="filter"
+      :isInstant="isInstant"
+      :hasClearButton="hasClearButton"
+    />
+  `,
+  range,
+  filter = createNextEditableNumberRangeFilter('age', range),
+  isInstant = false,
+  hasClearButton = true
+}: EditableNumberRangeFilterOptions = {}): EditableNumberRangeFilterAPI {
+  Vue.observable(filter);
+  const localVue = createLocalVue();
+  localVue.use(Vuex);
+  const store = new Store<DeepPartial<RootXStoreState>>({});
+  installNewXPlugin({ store }, localVue);
+
+  XPlugin.resetInstance();
+  XPlugin.registerXModule(facetsNextXModule);
```

Comment:
maybe move the clear getters cache to the `installNewXPlugin` util?

Reply:
You still need to reset the `instance`. What I did was moving the `XPlugin.resetInstance` call to the `installNewXPlugin` utility. Seems tests keep passing, So we can start removing those calls from the `spec` files.